CRM/Contact/BAO/Query.php

```diff
@@ -1489,6 +1489,15 @@ public static function convertFormValues(&$formValues, $wildcard = 0, $useEquals
           $params[] = array('on_hold', '=', $formValues['email_on_hold']['on_hold'], 0, 0);
         }
       }
+      elseif (substr($id, 0, 7) == 'custom_'
+        &&  (
+          substr($id, -9, 9) == '_relative'
+          || substr($id, -5, 5) == '_from'
+          || substr($id, -3, 3) == '_to'
+        )
+      ) {
+        self::convertCustomRelativeFields($formValues, $params, $values, $id);
+      }
```

Comment:
nope - but those lines above will format it in 4.6 - try searching on a custom date field that is not set up to search by range in 4.6 with your patch - the range custom fields are fine - but the ones that aren't do odd things with the date